server/liveblog/embed/templates/embed.html

```diff
@@ -1,64 +1,33 @@
 <!doctype html>
 <html ng-app="liveblog-embed">
     <head>
-        <title>Embed</title>
+        <title>{{ blog.title }}</title>
```

Comment:
yeah forgout about SEO.
not really what I had in mind with responsiveiframe but a good lib to use tho.
I was thinking more to interact with the embedable page, like adding in the title etc..